config/backstop/engine_scripts/cookies.json

```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "domain": ".www.yourdomain.com",
```

Comment:
Would an empty array in here be enough as I guess we don't need cookies for now? Also, 2027 is a long time away but what happens then? 😅